### PR TITLE
Fix menu placement

### PR DESCRIPTION
**Pull request by manuthecoder**
Fixes #7594 by changing the menu position

### CODE REVIEW

File: app/views/application/_support_button.html.erb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div data-controller="menu" data-menu-placement-value="<%= content_for?(:nav) ? "bottom-start" : "bottom-center" %>">
+<div data-controller="menu" data-menu-placement-value="bottom-start">
```
Comment (garyhtou) [inline app/views/application/_support_button.html.erb]: should this be `top-end`? I think `bottom-start` would technically put the menu below `?` button, and thus below the page's viewport

Comment (sampoder) [reply app/views/application/_support_button.html.erb]: `top-end` seems like the right thing to do, at least based on https://floating-ui.com/docs/tutorial#placements

**Comment by manu-gurudath** [comment]
Should we merge this?

**Comment by sampoder** [comment]
Does `top-end` fix it as well?

**Pull request merged**